firstvoices/backend/views/search_all_entries_views.py

```diff
@@ -658,6 +682,9 @@ def list(self, request, **kwargs):
         if search_params["visibility"] is None:
             return self.paginate_search_response(request, [], 0)
 
+        if search_params["sites"] is None:
+            return self.paginate_search_response(request, [], 0)
```

Comment:
Does this mean that `?sites=` will return no results?

Reply:
`search_params["sites"]` only returns None if all site slug are invalid, otherwise it returns "".  `?sites=` currently would return all entries, the same as how the types and visibility params work